sys/include/net/gnrc/nettype.h

```diff
@@ -48,6 +48,11 @@ extern "C" {
  * @note    Expand at will.
  */
 typedef enum {
+    /**
+     * @brief   TX synchronization data for passing up error data or
+     *          auxiliary data
+     */
+    GNRC_NETTYPE_TX_SYNC = -2,
```

Comment:
While reading [the release notes](https://github.com/RIOT-OS/RIOT/pull/15913), I started to wonder if it makes sense to set this to a different value than the [just removed `GNRC_NETTYPE_IOVEC`](https://github.com/RIOT-OS/RIOT/pull/15323). While it isn't exposed and should not be confusable with that removed value, some external testing scripts might still look out for this value. So maybe
```suggestion
    GNRC_NETTYPE_TX_SYNC = -3,
```

?